src/NLog.StructuredLogging.Json.Tests/NLog.StructuredLogging.Json.Tests.csproj

```diff
@@ -14,7 +14,7 @@
   </ItemGroup> 
   <ItemGroup>
     <PackageReference Include="FakeItEasy" Version="3.1.0" />
-    <PackageReference Include="Newtonsoft.Json" Version="10.0.1" />
+    <PackageReference Include="Newtonsoft.Json" Version="10.0.2" />
```

Comment:
FYI, references "flow" through to dependencies when using project references now, so the tests should only need to reference packages that the projects they are referencing _don't_ reference.